peachjam/js/components/FindDocuments/index.vue

```diff
@@ -518,9 +545,38 @@ export default {
       });
     },
 
+    formatFieldValues (key) {
+      if (key === 'date') return;
+      let formattedSearchString = '';
+      Object.keys(this.advancedFields[key]).forEach(fieldKey => {
+        const formattedFieldValue = this.advancedFields[key][fieldKey];
+        if (!formattedFieldValue) return;
+        let splitValue = formattedFieldValue.match(/[^\s"',]+|['"][^'"]*["']+/g);
+        if (fieldKey === 'all') {
+          splitValue = splitValue.join(' ');
+        } else if (fieldKey === 'exact') {
+          splitValue = splitValue.map(value => {
+            if (value.startsWith('"') || value.startsWith("'")) return value;
+            else return `"${value}"`;
+          });
+          splitValue = splitValue.join(' ');
+        } else if (fieldKey === 'any') {
+          splitValue = splitValue.join('|');
```

Comment:
I think this must all be wrapped in quotes, since the default operator is AND.

So: `foo (bar | baz)` rather than `foo bar | baz` which is actually equivalent to `(foo AND bar) or baz` because of the boolean precedence of AND and OR.

Reply:
In other words, if the user enters:

```
all: foo
any: bar baz
```

the search string should be: `foo (bar | baz)`